manipulate_pim_data/rule/general_information_on_rule_format.rst

```diff
@@ -302,6 +303,29 @@ Two parameters are required, two other are optional.
               items:
                 - t-shirts
 
+    Removing category "clothing" and its children will be as follows:
+
+    .. code-block:: yaml
+
+        actions:
+            - type: remove
+              field: categories
+              items:
+                - clothing
+              include_children: true
+
+    .. warning::
+
```

Comment:
Could you add also an example to remove from a whole tree (with the root category)?